apps/services/auth-api/infra/identity-server.ts

```diff
@@ -0,0 +1,140 @@
+import { service, ServiceBuilder, ref } from '../../../../infra/src/dsl/dsl'
+
+/**
+ * This setup is for the Identity Server, which is hosted in a different repository - https://github.com/island-is/identity-server.web
+ */
+export const serviceSetup = (services: {
+  authApi: ServiceBuilder<'services-auth-api'>
+}): ServiceBuilder<'identity-server'> => {
+  return service('identity-server')
+    .namespace('identity-server')
+    .image('identity-server')
+    .env({
+      AWS__CloudWatch__AuditLogGroup: '/identity-server/audit-log',
+      ASPNETCORE_URLS: 'http://*:5000',
+      CORECLR_ENABLE_PROFILING: '1',
+      CORECLR_PROFILER: '{846F5F1C-F9AE-4B07-969E-05C26BC060D8}',
+      CORECLR_PROFILER_PATH: '/opt/datadog/Datadog.Trace.ClrProfiler.Native.so',
+      DD_INTEGRATIONS: '/opt/datadog/integrations.json',
+      DD_DOTNET_TRACER_HOME: '/opt/datadog',
+      Datadog__Metrics__Port: '5003',
+      AudkenniSettings__Retries: '24',
+
+      AWS__SystemsManager__ParameterStore__DataProtectionPrefix: {
+        dev: '/k8s/identity-server/DataProtectionSecret',
+        staging: '/k8s/identity-server/DataProtectionSecret',
+        prod: '/k8s/identity-server/DataProtectionSecret',
+      },
+      CacheSettings__Enabled: {
+        dev: 'true',
+        staging: 'true',
+        prod: 'true',
+      },
+      CacheSettings__Memcached__Address: {
+        dev: 'identity-server.5fzau3.cfg.euw1.cache.amazonaws.com',
+        staging: 'identity-server.ab9ckb.cfg.euw1.cache.amazonaws.com',
+        prod: 'identity-server.dnugi2.cfg.euw1.cache.amazonaws.com',
+      },
+      CacheSettings__Memcached__Port: {
+        dev: '11211',
+        staging: '11211',
+        prod: '11211',
+      },
+      CacheSettings__Redis__Address: {
+        dev:
+          'clustercfg.general-redis-cluster-group.5fzau3.euw1.cache.amazonaws.com',
+        staging:
+          'clustercfg.general-redis-cluster-group.ab9ckb.euw1.cache.amazonaws.com',
+        prod:
+          'clustercfg.general-redis-cluster-group.dnugi2.euw1.cache.amazonaws.com',
+      },
+      CacheSettings__Redis__Port: {
+        dev: '6379',
+        staging: '6379',
+        prod: '6379',
+      },
+      IdentityServer__EnableFakeLogin: {
+        dev: 'true',
+        staging: 'true',
+        prod: 'true',
+      },
+      IdentityServer__EnableFeatureDeploymentWildcards: {
+        dev: 'true',
+        staging: 'true',
+        prod: 'https://innskra.island.is',
+      },
+      IdentityServer__EnableDelegation: {
```

Comment:
this is not used.

Reply:
Addressed in [5f4360b](https://github.com/island-is/island.is/pull/7350/commits/5f4360bbf4cc649aa87167141a3d743e60d4ac0c)